Travis CI build fails

<!-- Hello! If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on chat. You can ignore or delete this text, it is commented and won't appear when the issue is submitted or previewed.

Chat: https://gitter.im/coala/corobo
Documentation: https://github.com/coala/corobo/blob/master/docs/corobo.rst
-->
### Overview Description
The travis CI build for corobo is failing.
# Steps to Reproduce

1. Go to https://travis-ci.com
2. Run travis CI build for the coala/corobo repository

# Actual Results
Following are the errors occuring:
1.
```
tests/git_stats_test.py:4: in <module>
    from IGitt.GitHub.GitHubMergeRequest import GitHubMergeRequest
/usr/local/lib/python3.9/site-packages/IGitt/GitHub/__init__.py:13: in <module>
    from IGitt.Interfaces import Token, get, post
/usr/local/lib/python3.9/site-packages/IGitt/Interfaces/__init__.py:157: in <module>
    def get_response(method: Callable,
/usr/local/lib/python3.9/site-packages/backoff/_decorator.py:153: in decorate
    if asyncio.Task.current_task() is not None:
    AttributeError: type object '_asyncio.Task' has no attribute 'current_task'
```
2.
```
$ python -m pytest
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pluggy/manager.py", line 253, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2460, in load
    self.require(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2483, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (pytest 3.6.4 (/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages), Requirement.parse('pytest>=4.6'))
```
3.
```
$ python -m pytest
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --doctest-ignore-import-error
```
4.
```
self = <tests.wolfram_alpha_test.WolframAlphaTest testMethod=test_wa>
    @my_vcr.use_cassette('tests/cassettes/wa.yaml')
    def test_wa(self):
        self.push_message('!wa 2^6')
        self.assertIn('64', self.pop_message())
        with self.assertLogs() as cm:
            self.assertCommand('!wa this is a sentence',
                               'Dunno')
        self.assertIn('INFO:errbot.plugins.WolframAlpha:KeyError triggered on '
>                     'retrieving pods.', cm.output)
       AssertionError: 'INFO:errbot.plugins.WolframAlpha:KeyError triggered on retrieving pods.' not found in
['INFO:errbot.plugins.ACLs:Matching ACL {} against username gbin@localhost for command WolframAlpha:wa.',
'INFO:errbot.plugins.ACLs:Check if wa is admin only command.', 'INFO:errbot.core:Processing command "wa" with
parameters "this is a sentence" from gbin@localhost', 'ERROR:errbot.core:callback_message on deprecate_bot_prefixes
crashed.\nTraceback (most recent call last):\n  File "/usr/local/lib/python3.6/site-packages/errbot/core.py", line 111, in
_dispatch_to_plugins\n    getattr(plugin, method)(*args, **kwargs)\n  File "plugins/deprecate_bot_prefixes.py", line 15, in
callback_message\n    for deprecated_prefix in self.bot_config.BOT_DEPRECATED_PREFIXES:\nAttributeError:
\'ShallowConfig\' object has no attribute \'BOT_DEPRECATED_PREFIXES\'', 'INFO:vcr.stubs:Playing response for <Request
(GET) https://api.wolframalpha.com/v2/query?input=this+is+a+sentence&appid=None> from cassette',
'INFO:errbot.backends.test:\n\n\nMESSAGE:\nDunno :(\n\n\n']
tests/wolfram_alpha_test.py:22: AssertionError
```

# Expected Results
No errors. All tests should pass with 100% test coverage. All commands should exit with 0.

# Reproducibility
Every time

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3-alpine
+FROM python:3.6-alpine
 
 ENV BOT_ROOT=/opt/errbot
 

--- a/plugins/wolfram_alpha.py
+++ b/plugins/wolfram_alpha.py
@@ -24,11 +24,10 @@ class WolframAlpha(DefaultConfigMixin, BotPlugin):
         """
         ans = ''
         res = self.client.query(arg)
-        try:
-            for pod in res.pods:
-                if pod.title in ['Result', 'Results']:
-                    for sub in pod.subpods:
-                        ans += sub.plaintext
-        except AttributeError:
-            self.log.info('KeyError triggered on retrieving pods.')
+        for pod in res.pods:
+            if pod.title in ['Result', 'Results']:
+                for sub in pod.subpods:
+                    ans += sub.plaintext
+        if ans == '':
+            self.log.info("No results found!")
         return ans if ans else 'Dunno :('

--- a/setup.cfg
+++ b/setup.cfg
@@ -18,7 +18,7 @@ addopts =
   -p no:logging
   --color=yes
   --doctest-glob=*.rst
-  --doctest-ignore-import-error
+  --doctest-ignore-import-errors
   --error-for-skips
   --cov
   --instafail

--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -13,7 +13,7 @@ codecov~=2.0.5
 moban~=0.3.3 ; python_version > '3.0'
 packaging~=16.8
 pytest~=3.6.4
-pytest-cov~=2.4
+pytest-cov>=2.4,<=2.9
 pytest-env~=0.6.0
 pytest-error-for-skips~=1.0
 pytest-instafail~=0.3.0

--- a/tests/wolfram_alpha_test.py
+++ b/tests/wolfram_alpha_test.py
@@ -18,7 +18,7 @@ class WolframAlphaTest(IsolatedTestCase):
         with self.assertLogs() as cm:
             self.assertCommand('!wa this is a sentence',
                                'Dunno')
-        self.assertIn('INFO:errbot.plugins.WolframAlpha:KeyError triggered on '
-                      'retrieving pods.', cm.output)
+        self.assertIn('INFO:errbot.plugins.WolframAlpha:'
+                      'No results found!', cm.output)
         self.assertCommand('!plugin config WolframAlpha',
                            '{\'WA_TOKEN\': None}')

